api_specs/v2/component_analyses.yaml

```diff
@@ -1,17 +1,77 @@
 openapi: 3.0.1
 info:
   title: Analytics Core API Specs for Version 2
-  version: "2"
+  version: '2.1'
 servers:
 - url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
   description: "Dependency Analytics Server"
 security:
 - tokenAuth: []
 paths:
+  /component-analyses/:
+    post:
+      tags:
+        - CA batch call
+      summary: Get security and other details about a software component and recommendations to change to a more secured version in a batched manner.
+      parameters:
+        - name: uuid
+          in: header
+          description: User Identification uuid
+          required: false
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RequestBody'
+
+      responses:
+        200:
+          description: Detailed response for the requested application component batch
+          headers:
+            uuid:
+              schema:
+                type: string
+              description: User uuid.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/FreeUser'
+                    - $ref: '#/components/schemas/RegisteredUser'
+                    - $ref: '#/components/schemas/NoVulnerabilities'
+              examples:
+                FreeUser Batch:
+                  $ref: '#/components/examples/free-user-batch'
+                RegisteredUser Batch:
+                  $ref: '#/components/examples/registered-user-batch'
+                No Vulnerability Batch:
+                  $ref: '#/components/examples/no-vul-batch'
+
+        202:
+          description: All Packages are unknown. Triggers Ingestion.
```

Comment:
This description has to be changed to reflect the nature of this status code.

Reply:
Ack